Spoj1Divisor Summation/Program.cs

```diff
@@ -1,32 +1,82 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+
 
 namespace Spoj1Divisor_Summation
 {
-    public class Program
-    {
-        public static void Main()
-        {
-            var reader = new StreamReader(Console.OpenStandardInput());
-            var writer = new StreamWriter(Console.OpenStandardOutput());
+    public class Program
+    {
+        public const int MinRange = 1;
+        public const int MaxRange = 500000;
+
+        public static void Main()
+        {
+            var reader = new StreamReader(Console.OpenStandardInput());
+            var writer = new StreamWriter(Console.OpenStandardOutput());
             writer.AutoFlush = false;
 
-            DivisorSumation(reader, writer);
+            DivisorSumation(reader, writer);
         }
 
         public static void DivisorSumation(TextReader reader, TextWriter writer)
         {
-           //*******************************
-           //* TODO: Implement method body *
-           //*******************************
-
+            WriteInformationAboutData("Input");
+            ValidateAndCountDivisorSumationFromInputNumbers(reader, writer);
+            WriteInformationAboutData("Output");
             reader.Close();
             writer.Flush();
             writer.Close();
         }
+
+        private static void ValidateAndCountDivisorSumationFromInputNumbers(TextReader reader, TextWriter writer)
+        {
+            bool isNumber = false;
+
+            do
+            {
+                int amountOfInputs = 0;

```

Comment:
zamiast int i string można użyć var poprawia to czytelność bo zmienne są pod sobą a typ można wywnioskować lub użyć IDE